src/Misc/MoveToHecke.jl

```diff
@@ -16,6 +16,82 @@ end
 
 Oscar.canonical_unit(x::AbsSimpleNumFieldOrderQuoRingElem) = one(parent(x))
 
+function Hecke.numerator(f::QQPolyRingElem, parent::ZZPolyRing = Hecke.Globals.Zx)
+  g = parent()
+  ccall((:fmpq_poly_get_numerator, Nemo.libflint), Cvoid, (Ref{ZZPolyRingElem}, Ref{QQPolyRingElem}), g, f)
+  return g
+end
+
+Hecke.minpoly(a::QQBarFieldElem) = minpoly(Hecke.Globals.Qx, a)
+
+function extend_domain_to_fraction_field(phi::Map{<:MPolyRing, <:Ring})
+  ext_dom = fraction_field(domain(phi))
+  return MapFromFunc(ext_dom, codomain(phi), x->phi(numerator(x))*inv(phi(denominator(x))))
+end
+
+
+function iszero(P::EllipticCurvePoint)
+  return iszero(P[1]) && isone(P[2]) && iszero(P[3])
+end
+
+@doc raw"""
+    extended_ade(ADE::Symbol, n::Int)
+
+Return the dual intersection matrix of an extended ade Dynkin diagram
+as well as the isotropic vector (with positive coefficients in the roots).
+"""
+function extended_ade(ADE::Symbol, n::Int)
```

Comment:
Why should this be moved to Hecke? (Not objecting, but it is simply not clear to me)

Reply:
This was in a different file `MoveMeToHecke.jl`, maybe put there by @simonbrandhorst. I was just merging these two files